### ISSUE TITLE
Accessibility: meet 'multiple ways' requirement

### ISSUE THREAD

**Issue by markjefferson-gov** [ministryofjustice/find-moj-data#899]
An issue was highlighted in the [basic accessibility check](https://docs.google.com/spreadsheets/d/1MqZJFOHzdpo1KpNuWi3gDTxqmLv27NT90kxOBzLBrY0/edit?usp=sharing) with site navigation.

This relates to [WCAG 2.4.5 Multiple Ways](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html).

The advice from the accessibility team states: 

> "to be able to pass, you need to be able to access every other page directly from every page. So, from 'User Guide' or 'Contact Us', you would still want to navigate directly 'Home' instead of scrolling up to find the link within the body of the text. Technically, you can still navigate to 'Home', so we can't say 100% that you would fail on this as it stands, but it would be great for consistency and ease of use if you had a navigation menu at the top which matched the other pages. Another option is to have a sitemap in the footer of each page, but would only be a suggestion."

One solution would be to bring the user guide and contact us onto Find MoJ forms. This would be the best solution for usability in general as it makes navigating the site easier and more seamless.

But we could also consider creating a simple site map (e.g. [we built this on my previous project](https://moj-forms.service.justice.gov.uk/sitemap/)). This would need to be in the footer of all pages, including the user guide.

**Comment by teeceeas** [comment]
Add this to the accessibility audit as it requires more clarity

**Comment by DeanWillicombe82** [comment]
Links to bringing the User Guide into the main service - this will then resolve.
